src/script/conversation/ConversationRepository.js

```diff
@@ -3178,6 +3178,15 @@ export class ConversationRepository {
           z.error.ConversationError.TYPE.CONVERSATION_NOT_FOUND,
         ];
 
+        const isConversationNotFound = error.label === BackendClientError.LABEL.ACCESS_DENIED;
```

Comment:
Let's use a different name to not confuse access denied with a real conversation not found error. How about `isRemovedFromConversation`?